eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/health/CheckResultCache.java

```diff
@@ -41,8 +41,10 @@ public void update(String type, Long typeId, HealthCheckStatus status, String re
             cacheMap.put(type, subMap);
         }
         CheckResult oldResult = subMap.get(typeId);
-        String oldDesc = Objects.isNull(oldResult.getResultDesc()) ? "" : oldResult.getResultDesc() + "\n";
-        CheckResult result = new CheckResult(status, oldDesc + resultDesc, LocalDateTime.now(),
+        String oldDesc = Objects.isNull(oldResult.getResultDesc()) || oldResult.getResultDesc().isEmpty() ? "" : oldResult.getResultDesc() + "\n";
```

Comment:
Why `oldDesc` should have a `\n` at the end?

Reply:
` newdesc` is added after `oldDesc` , just to distinguish

Reply:
You mean like this?

```
oldDesc
newdesc
```

By the way, `Objects.isNull` is intended for use within Java 8 lambda filtering. Just keep the code simple using `whatever != null` will be better.

Reply:
```
oldDesc
newdesc
```
that's what I mean

Reply:
Then such writting will be better:

```
String whatever = oldDesc + Constants.NEW_LINE_ENDING + newDesc;
```

What do you think?

Reply:
okay

Reply:
Well, using a constant is meant to split `oldDesc` and `newDesc` strings. So the following is better:

```
String description = oldDesc + Constants.NEW_LINE_ENDING + resultDesc;
```

Also, a constant can be referenced by its class name to indicate its usage field, so a `HEALTH_` prefix is unnecessary.

Reply:
if `oldDesc` is an empty string `""`, this add an `NEW_LINE_ENDING` is the beginning of the `description`.

Reply:
if-else is acceptable~